mllm/models/qwen3_5/configuration_qwen3_5.hpp

```
@@ -0,0 +1,161 @@
+  float rms_norm_eps = 1e-06;
+  int32_t vocab_size = 248320;
+
+  // Qwen3.5-specific: hybrid attention
+  bool attn_output_gate = true;
+  int32_t full_attention_interval = 4;
+  std::vector<std::string> layer_types;  // "full_attention" or "linear_attention"
+
+  // Qwen3.5-specific: partial RoPE
+  float partial_rotary_factor = 0.25;
+  float rope_theta = 10000000.0;
+  int32_t rotary_dim() const { return static_cast<int32_t>(head_dim * partial_rotary_factor); }
```

Clang (14.0.6)

[error] function 'rotary_dim' should be marked [[nodiscard]]

(modernize-use-nodiscard,-warnings-as-errors)